index.bs

```diff
@@ -13619,7 +13623,7 @@ Additionally, [=legacy platform objects=] have internal methods as defined in:
                 1.  Return <emu-val>true</emu-val>.
     1.  If |O| does not implement an [=interface=] with the [{{Global}}] [=extended attribute=],
         then set |Desc|.\[[Configurable]] to <emu-val>true</emu-val>.
-    1.  Return <a abstract-op>OrdinaryDefineOwnProperty</a>(|O|, |P|, |Desc|).
+    1.  Return [=?=] <a abstract-op>OrdinaryDefineOwnProperty</a>(|O|, |P|, |Desc|).
```

Comment:
Can't fail because this does not have an custom throwy [[IsExtensible]] method and the custom [[GetOwnProperty]] method doesn't throw.

Reply:
Fixed.